src/Main.elm

```diff
@@ -0,0 +1,109 @@
+module Main exposing (Model, Msg(..), init, main, subscriptions, update, view)
+
+import Browser
+import Browser.Events
+import Html exposing (..)
+import Html.Attributes as Attributes exposing (style)
+import Html.Events exposing (onClick)
+import Json.Decode
+import Robot exposing (Model, Msg, keyDown, keyUp, moveRobot, view)
+
+
+main : Program () Model Msg
+main =
+    Browser.element
+        { init = init
+        , view = view
+        , update = update
+        , subscriptions = subscriptions
+        }
+
+
+type alias Model =
+    { robot1 : Robot.Model
+    , robot2 : Robot.Model
+    }
+
+
+type Msg
+    = Robot1Msg Robot.Msg
+    | Robot2Msg Robot.Msg
+    | KeyDown String
+    | KeyUp String
+
+
+init : () -> ( Model, Cmd Msg )
+init _ =
+    ( { robot1 =
+            { x = 900
+            , y = 500
+            , upKey = "ArrowUp"
+            , downKey = "ArrowDown"
+            , rightKey = "ArrowRight"
+            , leftKey = "ArrowLeft"
+            , color = "blue"
+            , speed = 5
+            , rotation = 0
+            , angle = 0
+            , velocity = 0
+            }
```

Comment:
This breaks the abstraction - `Robot` should be used as a black box.
Use `Robot.init` instead.